src/config.c

```diff
@@ -2995,6 +3007,7 @@ standardConfig static_configs[] = {
     createStringConfig("proc-title-template", NULL, MODIFIABLE_CONFIG, ALLOW_EMPTY_STRING, server.proc_title_template, CONFIG_DEFAULT_PROC_TITLE_TEMPLATE, isValidProcTitleTemplate, updateProcTitleTemplate),
     createStringConfig("bind-source-addr", NULL, MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.bind_source_addr, NULL, NULL, NULL),
     createStringConfig("logfile", NULL, IMMUTABLE_CONFIG, ALLOW_EMPTY_STRING, server.logfile, "", NULL, NULL),
+    createStringConfig("locale", NULL, MODIFIABLE_CONFIG, EMPTY_STRING_IS_NULL, server.locale, "", isValidLocale, NULL),
```

Comment:
`ALLOW_EMPTY_STRING` should be used instead of `EMPTY_STRING_IS_NULL`.
according to the doc in https://man7.org/linux/man-pages/man3/setlocale.3.html
```
If locale is NULL, the current locale is only queried, not modified.
```

Reply:
That's right, we don't need to support queries.